src/fiatExchanges/ExternalExchanges.tsx

```diff
@@ -71,7 +71,10 @@ function ExternalExchanges({ route }: Props) {
         <>
           <Text style={styles.pleaseSelectExchange}>
             {t('youCanTransferIn', {
-              digitalAsset: CURRENCIES[route.params.currency].cashTag,
```

Comment:
I see why you're not putting cREAL in the CURRENCIES object, since that has bigger consequences across the app, in particular making cREAL transactions visible in the home screen.

However, it would be nice if there was a todo list of cleanup that we need to do as a fast-follow to this PR, otherwise I worry we'll end up with a bunch of tech debt and a fading memory of what we actually need to do to address it. 

Maybe a Linear project like "enable cREAL across the app" and an issue within it that says "make cREAL visible in the transaction feed on the home screen", and includes the implementation detail "add cREAL as a key to the CURRENCY object in `currencies.ts` in the wallet